package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "wannabe-deviant-art",
+  "version": "1.0.0",
+  "description": "A wanna be deviant art web application.",
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CloeD/wannabe-deviant-art.git"
+  },
+  "author": "Cloé Dutil, Jessy Anglehart-Nunes",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/CloeD/wannabe-deviant-art/issues"
+  },
+  "homepage": "https://github.com/CloeD/wannabe-deviant-art#readme",
```

Comment:
You should do a readme when you have some time

Reply:
Will be done in #3